package.json

```
@@ -94,7 +94,21 @@
         "command": "fluttergpt.fixErrors.menu",
         "title": "Fix Errors",
         "when": "false"
+      },
+      {
+        "command": "fluttergpt.createResponsiveWidgetFromCode",
+        "title": "FlutterGPT Create: Create Responsive Widget from Code"
+      },
+      {
+        "command": "fluttergpt.createResponsiveWidgetFromCode.menu",
+        "title": "Create Responsive Widget from Code",
+        "when": "false"
+      },
+      {
+        "command": "fluttergpt.createResponsiveWidgetFromDescription",
+        "title": "[FlutterGPT] Responsive Widget from Description"
```

```suggestion
        "title": "FlutterGPT Create: Responsive Widget from Description"
```
Please keep naming consistent to other commands.